filebeat/_meta/fields.common.yml

```diff
@@ -108,6 +108,11 @@
       description: >
         Logging level.
 
+    - name: log.truncated
```

Comment:
Default type is `keyword`. I assume this needs to be integer/long?

I like that it's not just a bool but a value that tells how much was deleted.